var/spack/repos/builtin/packages/fish/package.py

```diff
@@ -30,7 +32,7 @@ class Fish(CMakePackage):
     variant("docs", default=False, description="Build documentation")
 
     # https://github.com/fish-shell/fish-shell#dependencies-1
-    depends_on("cmake@3.2:", type="build")
+    depends_on("cmake@3.5:", type="build")
```

Comment:
This is only true for newer versions. Do you know when they bumped the CMake requirement?

Reply:
https://github.com/fish-shell/fish-shell/commit/210dda2c4c0701658c2ad79fc65d813049ec1900. Looks like a few days after 3.3.1 was released.

Reply:
Cool, then it should be:
```suggestion
    depends_on("cmake@3.5:", when="@3.4:", type="build")
    depends_on("cmake@3.2:", type="build")
```